src/Crate/PDO/PDO.php

```diff
@@ -22,49 +22,59 @@
 
 namespace Crate\PDO;
 
+use Crate\PDO\Exception\InvalidArgumentException;
 use Crate\PDO\Exception\PDOException;
+use Crate\PDO\Http\ServerPool;
 use Crate\Stdlib\ArrayUtils;
 use PDO as BasePDO;
 
 class PDO extends BasePDO implements PDOInterface
 {
-    const VERSION = '0.8.0';
-    const DRIVER_NAME = 'crate';
-
-    const DSN_REGEX = '/^(?:crate:)(?:((?:[\w\d\.-]+:\d+\,?)+))\/?([\w]+)?$/';
-
-    const CRATE_ATTR_HTTP_BASIC_AUTH    = 1000;
-    /**
-     * deprecated since version 0.4
-     * @deprecated
-     */
-    const ATTR_HTTP_BASIC_AUTH          = self::CRATE_ATTR_HTTP_BASIC_AUTH;
-    const CRATE_ATTR_DEFAULT_SCHEMA     = 1001;
-
-    const PARAM_FLOAT       = 6;
-    const PARAM_DOUBLE      = 7;
-    const PARAM_LONG        = 8;
-    const PARAM_ARRAY       = 9;
-    const PARAM_OBJECT      = 10;
-    const PARAM_TIMESTAMP   = 11;
-    const PARAM_IP          = 12;
+    public const VERSION     = '0.8.0';
+    public const DRIVER_NAME = 'crate';
+
+    public const DSN_REGEX = '/^(?:crate:)(?:((?:[\w\.-]+:\d+\,?)+))\/?([\w]+)?$/';
+
+    public const CRATE_ATTR_HTTP_BASIC_AUTH = 1000;
```

Comment:
Are the constant values referencing existing PDO constants, or did you define something yourself on here?